tsdfileapi/api.py

```diff
@@ -126,11 +141,37 @@ def set_config() -> None:
     if 'accept_calls_per_event_loop' in _config:
         tornado.netutil.ACCEPT_CALLS_PER_EVENT_LOOP = _config['accept_calls_per_event_loop']
     options.logging = _config.get('log_level', 'info')
+    try:
+        projects_pool = postgres_init(
+            {
+                'user': _config.get('iamdb_user'),
+                'pw': _config.get('iamdb_pw'),
+                'host': _config.get('iamdb_host'),
+                'dbname': _config.get('iamdb_dbname'),
+            }
+        )
+    except Exception as e:
```

Comment:
Could be a good idea to log the caught error, but I am guessing that in this context -- during configuration phase here, evidently -- it is entirely benign and thus doesn't need logging?

Reply:
No logging needed no